getting-started/repl/intergration.md

```diff
@@ -41,32 +41,32 @@ Most code examples in the Telerik UI for Blazor documentation are directly runna
 
 Telerik Extensions for [Visual Studio]({%slug getting-started-vs-integration-overview%}) and [Visual Studio Code]({%slug getting-started-vs-code-integration-overview%}) provide option to easily share your code to the Telerik REPL for Blazor with a few clicks. You can share the whole file or just a selection.
 
-The command can be used if more than 10 symbols are selected. Otherwise Visual Studio will disable the command, and Visual Studio Code will show an error message ("Expected at least 10 characters"). In case of multiple selection every new piece will be concatenated on a new line.
+The command can be used if more than 10 symbols are selected. Otherwise Visual Studio will disable the command, and Visual Studio Code will show an error message ("Expected at least 10 characters"). In case of multiple selection, every new piece will be concatenated on a new line.
 
 If no selection is made, the whole file will be shared to Telerik REPL for Blazor. Only single files can be shared per command and not the whole application. Once the share to REPL process is triggered, status notifications will appear in the bottom-right corner for a few seconds. After that, the default browser will open with the uploaded snippet.
 
-The `Share to Telerik REPL for Blazor` command can be accessed as follows:
+### Sharing to REPL in Visual Studio
 
-### Visual Studio
+1. Get the extension from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=TelerikInc.TelerikBlazorVSExtensions). 
```

Comment:
There are two separate links for VS 2022 and VS 2019. I suggest to link the dedicated page.

```suggestion
1. [Get the extension from the Visual Studio Marketplace]({%slug getting-started-vs-integration-overview%}). 
```

Reply:
I will add the two links here to avoid complicating the steps for the readers and making them go through another article first.
```suggestion
1. Get the extension from the Visual Studio Marketplace:

   * <a href="https://marketplace.visualstudio.com/items?itemName=TelerikInc.ProgressTelerikBlazorVSExtensions" target="_blank">Extension for Visual Studio 2022 for Windows</a>
   * <a href="https://marketplace.visualstudio.com/items?itemName=TelerikInc.TelerikBlazorVSExtensions" target="_blank">Extension for Visual Studio 2019 for Windows</a>
```